src/connectors/configure/connectConfigure.js

```diff
@@ -1,18 +1,21 @@
-import isFunction from 'lodash/isFunction';
+import noop from 'lodash/noop';
 import isPlainObject from 'lodash/isPlainObject';
 
 import { enhanceConfiguration } from '../../lib/InstantSearch.js';
 
 const usage = `Usage:
-var customConfigureWidget = connectConfigure(
+var customConfigure = connectConfigure(
   function renderFn(params, isFirstRendering) {
     // params = {
     //   refine,
-    //   widgetParams
+    //   widgetParams,
     // }
-  },
-  function disposeFn() {}
-)
+  }
+);
+search.addWidget(
+  customConfigure({})
```

Comment:
We should add `searchParameters` in the usage because it's a required parameter.